shell/lib/db.js

```diff
@@ -321,6 +331,18 @@ isAdmin = function() {
   }
 }
 
+isAdminById = function(id) {
+  check(id, String);
+  // Returns true if the user's id is the administrator.
+
+  var user = Meteor.users.findOne({_id: id})
```

Comment:
use `{fields: {isAdmin: 1}}` to select just the `isAdmin` field. (See examples in `stats.js` -- you could probably change them to use this function, in fact.)